search.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
No need to add this file here

Reply:
This is a local test file. Let me exclude it from the PR